samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -0,0 +1,33 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Could you update this template.yaml following this new one for dotnet?

https://github.com/awslabs/aws-sam-cli/blob/0622b9bb9b601a6c8c1e4054c2a0c93c2e70c96b/samcli/local/init/templates/cookiecutter-aws-sam-hello-dotnet/%7B%7Bcookiecutter.project_name%7D%7D/template.yaml

Essentially, it removes Env Variables (not being used) and adds additional comments in the Outputs

Reply:
Looking at the new file, I don't think I need to change this as I don't include the API in this example (the 50 second warm up time based on module loading makes that prohibitive), let me know if you tihnk anything specific is missing from my file though and ill change it.